src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -10,7 +10,6 @@ import compose from '../../compose';
 import * as PersonalDetails from '../../actions/PersonalDetails';
 import * as Pusher from '../../Pusher/pusher';
 import PusherConnectionManager from '../../PusherConnectionManager';
-import UnreadIndicatorUpdater from '../../UnreadIndicatorUpdater';
 import ROUTES from '../../../ROUTES';
```

Comment:
```suggestion
import '../../UnreadIndicatorUpdater/index';
import ROUTES from '../../../ROUTES';
```

@tgolen I guess the subscription should be imported somewhere in the App

Reply:
OK, sorry. I think I might have broken this when I just fixed the linter tests via the GitHub web interface. Sorry about that! Let me do this properly and test it out again.

Reply:
changes looks good